docs/changes/visualization/16347.feature.rst

```diff
@@ -0,0 +1 @@
+Add support for custom coordinate frames for ``WCSAxes`` through a context manager ``astropy.visualization.wcsaxes.custom_ucd_wcscoord_mapping`` using a custom property mapping ``CUSTOM_UCD_COORD_META_MAPPING`` in ``astropy.visualization.wcsaxes.wcsapi``.
```

Comment:
Should we maybe not mention the dictionary here since it's an implementation detail?